pan3d/explorers/slice_explorer.py

```diff
@@ -529,6 +615,39 @@ def _build_ui(self):
                             items=("data_vars", self.vars),
                         )
 
+                with v3.VCard():
+                    with v3.VCardTitle():
+                        html.Div("Scaling")
+                    with v3.VCardText():
+                        with v3.VRow():
+                            with v3.VCol(
+                                cols=4,
+                                classes="text-center font-weight-medium d-flex flex-column",
+                            ):
+                                html.Div("{{slice_dims[0]}}")
+                                NumericField(
+                                    model_value=("x_scale",),
+                                    update_event=partial(self.on_axis_scale_change, 0),
```

Comment:
I don't like the use of partial here

Reply:
but for simplicity it is probably ok... Otherwise I was thinking you would just pass what trame expect `update_event=(self. on_axis_scale_change, "[0, Number($event.target.value)]")`